go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/timescale/tsbs
 
-go 1.12
+go 1.13
```

Comment:
I don't think is actually a good change (we might want to drop it to `1.11` even). This is supposed to be the minimum version of Go that this module works with, and afaik there isn't any reason people with `1.12` shouldn't be able to build it. That is, we aren't depending on any `1.13` specific APIs